File: src/TheTeacher.Infrastructure/Exceptions/ServiceErrorCodes.cs
namespace TheTeacher.Infrastructure.Exceptions
{
    public class ServiceErrorCodes
    {
        public static string EmailInUse => "email_in_use";
        public static string TeacherAlreadyExists => "teacher_already_exists";
        public static string UserAlreadyExists => "user_already_exists";        
        public static string TeacherNotFound => "teacher_not_found";
        public static string UserNotFound => "user_not_found";
        public static string InvalidCredentials => "invalid_credentials";
        public static string InvalidEmail => "invalid_email";
        public static string InvalidSubjectDetails => "invalid_subject_details";
        public static string InvalidNewPassword => "invalid_new_password";
    }
}

File: src/TheTeacher.Web/src/app/exception/service-error-codes.ts
export class ServiceErrorCodes {
    static readonly emailInUse: string = "email_in_use";
    static readonly teacherAlreadyExists: string = "teacher_already_exists";
    static readonly userAlreadyExists: string = "user_already_exists";        
    static readonly teacherNotFound: string = "teacher_not_found";
    static readonly userNotFound: string = "user_not_found";
    static readonly invalidCredentials: string = "invalid_credentials";
    static readonly invalidEmail: string = "invalid_email";
    static readonly invalidSubjectDetails: string = "invalid_subject_details";
    static readonly invalidNewPassword: string = "invalid_new_password";
}